client/my-sites/plans/jetpack-plans/jetpack-complete-page/items-included/style.scss

```diff
@@ -0,0 +1,63 @@
+.single-product__wrapper {
+	display: inline-flex;
+	align-items: center;
+}
+
+.single-product__icon {
+	width: 2.4rem;
+	height: 2.4rem;
+	background: linear-gradient(159.87deg, #f6f6f4 7.24%, #f7f4ea 64.73%, #ddedd5 116.53%);
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.single-product__text {
+	font-weight: 600;
+	font-style: normal;
+	margin-left: 0.5rem;
+	font-size: 0.875rem;
+}
+
+.single-product__info {
+	margin-left: 0.75rem;
+
+	button {
+		display: flex;
+	}
+}
+
+.single-product__info-popover-wrapper {
+	color: var(--color-neutral-100);
+
+	hr {
+		margin: 0.75rem 0;
+	}
+}
+
+.single-product__info-link {
+	font-style: normal;
+	font-weight: 400;
+	line-height: 24px;
+
+	display: flex;
+	align-items: center;
+	letter-spacing: -0.01em;
+	text-decoration-line: underline;
+
+	color: var(--color-neutral-100) !important;
+}
+
+.items-included__list-table {
+	column-gap: 40px;
+	display: grid;
+	list-style-type: none;
+	margin: 0;
+	row-gap: 1rem;
+}
+
+@media (min-width: 782px) {
+	.items-included__list-table {
+		grid-template-columns: repeat(2, 1fr);
+	}
+}
```

Comment:
In Calypso, for css breakpoints we typically `@import` the mixins located either in `@wordpress/base-styles/mixins` (AFIK, These seems to be "desktop-first" breakpoints) or `@import` the mixins at `"calypso/assets/stylesheets/shared/mixins/breakpoints"` (which seems to allow you to do "mobile-first" breakpoints, but this approach is deprecated for some reason (I don't know why, but is still used within the Calypso codebase).  You would then also `@import` the `"@wordpress/base-styles/breakpoints"` which provide all the most used breakpoint variables (such as `$break-medium: 782px;`)
However this may just be a nit-pick because you only have one(1) breakpoint in this file, and therefore importing and using the mixins may just be overkill in this situation.. Probably is..
I'm not completely sure if there is a true convention or standard we need to be following concerning this and it's probably fine the way it is. I just wanted to make sure you were aware of the breakpoint mixins that are often used throughout Calypso. 😉

Reply:
Thank you, I was not aware with this.